client/systemtrayicon.cpp

```diff
@@ -89,3 +106,12 @@ void SystemTrayIcon::showHide()
         m_parent->setFocus();
     }
 }
+
+void SystemTrayIcon::focusChanged(QWidget* old, QWidget* now)
```

Comment:
```suggestion
void SystemTrayIcon::focusChanged(QWidget* old)
```
A nice thing about Qt slots is that you don't need to fully match the signal parameters; unused parameters at the end can be dropped.